docs/hack_operator_make_run.md

```diff
@@ -0,0 +1,274 @@
+# Hacking on Loki Operator using `make run`
+
+This document demonstrates how to use Loki Operator for development and testing locally on Kind and OpenShift using the `make run` command.
+
+_Note:_ This is helpful when you don't want to deploy the Loki Operator image everytime you make slight changes to the operator code.
+
+## Hacking on Loki Operator using kind
+
+[kind](https://kind.sigs.k8s.io/docs/user/quick-start/) is a tool for running local Kubernetes clusters using Docker container "nodes". kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+
+### Requirements
+
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) or [Openshift CLI](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html) for communicating with the cluster. The guide below will be using `kubectl` for the same.
+* Create a running Kubernetes cluster using kind.
+
+### Installation of Loki Operator
+
+* Install the CRDs into the cluster:
+
+  ```console
+  make install
+  ```
+
+  This will create a custom resource definition with the name `lokistacks.loki.openshift.io` which can be verified using:
+
+  ```console
+  kubectl get crd
```

Comment:
This is verification phase .... you need to limit the CRD's just to the `lokistacks.loki.openshift.io` ones